core/src/test/java/org/elasticsearch/search/aggregations/InternalAggregationTestCase.java

```diff
@@ -122,4 +135,56 @@ protected final T createUnmappedInstance(String name) {
     protected NamedWriteableRegistry getNamedWriteableRegistry() {
         return namedWriteableRegistry;
     }
+
+    protected ContextParser<Object, Aggregation> instanceParser() {
+        throw new UnsupportedOperationException("Not implemented yet");
+    }
```

Comment:
Sorry, my bad. What I meant was overwriting `xContentRegistry()` from ESTestCase, I mixed that up. That way you can e.g. provide the parser that you need in the test without the need of instanceParser(). I don't mind either way, feel free to use it or not.